src/libs/ajax/Apps.js

```diff
@@ -0,0 +1,63 @@
+import _ from 'lodash/fp'
+import * as qs from 'qs'
+import { appIdentifier, authOpts, fetchLeo, jsonBody } from 'src/libs/ajax'
```

Comment:
this pattern is putting a circular dependency in place...

Reply:
hmm.... the transpiler seems to handle this fine. I'll make an `ajax-common.js` for now to avoid this though

Reply:
The transpiler does handle it many times but FWIW at some point I ran into a difficult-to-understand error message, whose root cause turned out to be a circular dependency. Details escape me but I believe it was happening because some code line to execute depended on another which depended on the former.

Reply:
yeah, we are probably dodging the bullet here because the key exports involved have deferred usage... but generally good to just avoid this entirely.